app/src/ui/repositories-list/repository-list-item.tsx

```diff
@@ -94,7 +91,7 @@ export class RepositoryListItem extends React.Component<
       : path
 
     let prefix: string | null = null
-    if (this.props.needsDisambiguation && gitHubRepo) {
+    if (gitHubRepo) {
       prefix = `${gitHubRepo.owner.login}/`
```

Comment:
This prefix is used to show the owner in a slightly lighter colour. I'm not sure how the "never disambiguate" change here should impact this:

<img width="275" src="https://user-images.githubusercontent.com/359239/42759468-7accaa54-88dd-11e8-81f6-2911ce9f1a71.png">